5.1.I_have_no_vinaigrette.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+import random
+import numpy
+
+
+def pick_random_date_in_range(first_date, last_date):
+    """
+     Returns a random date between first_date and last_date.
+    :param first_date: The first date in the range.
+    :param last_date: The last date in the range.
+    :return: A random date between first_date and last_date.
+    """
+    first_date, last_date = (first_date, last_date) if first_date <= last_date else (last_date, first_date)
```

Comment:
I would simplify to `first_date, last_date = min(first_date, last_date), max(first_date, last_date).`